index.src.html

```diff
@@ -504,64 +491,28 @@
-
-This structure contains additional data required to verify the attestation signature.
+<div dfn-for="WebAuthnAttestation">
+    The <dfn>type</dfn> member specifies the type of attestation statement contained in this structure. This specification
+    defines a number of attestation types, in [[#defined-attestation-types]]. Other attestation types may be defined in later
+    versions of this specification.
 
-<div dfn-for="AttestationHeader">
-    The <dfn>claimedAAGUID</dfn> element contains the claimed Authenticator Attestation GUID (a version 4 GUID, see
-    [[RFC4122]]). This value is used by the <a>[RP]</a> to look up the trust anchor for verifying the following
-    {{AttestationCore}} object. If the verification succeeds, the AAGUID related to the trust anchor is trusted. This field MUST
-    be present, if either no attestation certificates are used (e.g., DAA) or if the attestation certificate doesn't contain the
-    AAGUID value in an extension.
-
-    The <dfn>x5c</dfn> attribute contains the attestation certificate and its certificate chain as described in [[!RFC7515]]
-    section 4.1.6.
+    The <dfn>clientData</dfn> member contains the base64url encoding of the parameters sent to the authenticator by the client.
+    (See <a>clientDataJSON</a> in [[#sec-client-data]].)
 
-    The <dfn>alg</dfn> element contains the name of the algorithm used to generate the attestation signature according to
-    [[!RFC7518]]. See [[#packed-attestation-signature]] for the signature algorithms to be implemented by servers.
+    The <dfn>statement</dfn> element contains the actual attestation statement. The structure of this object depends on the
+    attestation type. For more details, see [[#attestation]].
 </div>
 
-
-## Attestation core data (interface <dfn interface>AttestationCore</dfn>) ## {#sec-attestationcore}
-
-This structure contains the data attested by the Authenticator and a description of its structure. Different types of
-Authenticators might generate different object types (identified by {{AttestationCore/type}} and {{AttestationCore/version}}).
-
-<div dfn-for="AttestationCore">
-    The <dfn>type</dfn> member specifies the type of the rawData object. This specification defines a number of attestation raw
-    data types, in [[#defined-attestation-raw-data-types]]. Other attestation raw data types may be defined in further versions
-    of this specification.
-
-    The <dfn>version</dfn> member specifies the version number of the rawData object.
-
-    The <dfn>rawData</dfn> object contains the attested public key and the |clientDataHash|.
-
-    The <dfn>clientData</dfn> member contains the base64url encoding of <a>clientDataJSON</a> (see [[#signature-format]]). The
-    exact encoding must be preserved as the hash (clientDataHash) has been computed over it.
-</div>
+This attestation statement is delivered to the <a>[RP]</a> by the script, using methods outside the scope of this specification.
```

Comment:
s/ <a>[RP]</a> by the script / <a>[RP]</a> server by the RP's script /

see also issue #80.